### ISSUE TITLE
the need to include glider-compat.min.js is not clearly mentioned in documentation

### ISSUE THREAD

**Issue by JackieHayduk** [NickPiscitelli/Glider.js#118]
The documentation could use a more clear direction around the compatibility file needing to be included. I was debugging an IE11 issue and was originally thinking our version of your plugin was out of date and then I stumbled across the now closed issue that this file was not in the repo. By including it my IE issue was resolved. I would add a note about browser compatibility in your getting started info for future users.

**Comment by gobaSportsGroup** [comment]
Hey @JackieHayduk can you share with me how you resolved the IE11 issue?

**Comment by JackieHayduk** [comment]
@gobaSportsGroup In the file directory there is a file called glider-compat.min.js which isn't included by default. If importing the plugin at the top of a js file as such: 
`import Glider from 'glider-js'; `

you have to also include an import to the compat file specifically for me the path was this:
`import '../../../../../node_modules/glider-js/glider-compat.min.js';`

so my complete import for this plugin is:
`import Glider from 'glider-js';
import '../../../../../node_modules/glider-js/glider-compat.min.js';`

Hope this helps!

**Comment by NickPiscitelli** [comment]
The compatibility file is documented on the README in the repository, but it is indeed missing from the documentation.

**Comment by sardug10** [comment]
Hey, if this issue is still open, can I work on it?

**Comment by NickPiscitelli** [comment]
Yes, feel free to work on any open issues. Feel free to email me if you
need any support. Thank you.

**Comment by NickPiscitelli** [comment]
Sorry for the late reply.

After further review of this issue, both the documentation and the README already mention this compatibility file in the browser support section. I'm not sure if calling any extra attention to this file is warranted, but if so, it should be listed as a footnote and not as a step of the install process. Ultimately, few users will need this file since market share for legacy browsers is quite insignificant as of late.
